### ISSUE TITLE
Disabled Pager buttons in Grid do not have aria-disabled attribute set

### ISSUE THREAD

**Issue by veselints** [telerik/kendo-ui-core#6775]
### Bug report
Disabled Pager buttons in Grid do not have aria-disabled attribute set

### Reproduction of the problem
Could be observed on the Grid KB navigation demo: https://demos.telerik.com/kendo-ui/grid/keyboard-navigation

### Expected/desired behavior
`aria-disabled="true"` should be set on disabled Pager buttons.

### TicketID:
1558582

### Environment
* **Kendo UI version:** 2022.1.301
* **Browser:** all
